root/programs/C#/3_Build_Framework47.bat

```diff
@@ -0,0 +1,39 @@
+setlocal
```

Comment:
@daisukenishino 

The user might not understand the meaning of *47*.
It might be a good idea to name this file *3_Build_Framework_net47.bat*.